devops/jobs/CreatePingdomAlerts.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+
+    Variables without defaults are marked (required)
+
+    Variables consumed for this job:
+        * NOTIFY_ON_FAILURE: (Required - email address)
+        * SECURE_GIT_CREDENTIALS: (Required - jenkins name of git credentials)
+        * PINGDOM_EMAIL: (Required - The Pingdom Email)
+        * PINGDOM_API_KEY: (Required - Pingdom API Key)
+        * PINGDOM_PASSWORD: (Required - Pingdom Password)
+        * PINGDOM_ALERT_CONFIG_FILE (The pingdom Alert config file)
+
+*/
+package devops.jobs
+
+
+import static org.edx.jenkins.dsl.DevopsConstants.merge_to_master_trigger
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+import static org.edx.jenkins.dsl.Constants.common_logrotator
+
+
+class CreatePingdomAlerts {
+    public static def job = { dslFactory, extraVars ->
+        assert extraVars.containsKey('PINGDOM_EMAIL') : "Required PINGDOM_EMAIL setting missing from configuration"
+        assert extraVars.containsKey('PINGDOM_ALERT_CONFIG_FILE') : "Required pingdom alert config file(PINGDOM_ALERT_CONFIG_FILE) missing from configuration"
+        dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/create_pingdom_alerts") {
+            assert extraVars.containsKey('CONFIGURATION_REPO_URL') : "Please define CONFIGURATION_REPO_URL"
```

Comment:
Please document this with the other vars above, and note from some of the other webhook trigger jobs which format of URL you need (I believe it's documented in the DockerCI jobs)